packages/lab/src/components/Flow/Empty/Empty.tsx

```diff
@@ -0,0 +1,37 @@
+import { css } from "@emotion/css";
+import {
+  HvEmptyState,
+  HvEmptyStateProps,
+} from "@hitachivantara/uikit-react-core";
+import { Fail } from "@hitachivantara/uikit-react-icons";
+import { theme } from "@hitachivantara/uikit-styles";
+import { useStore } from "reactflow";
+
+export interface HvFlowEmptyProps {
```

Comment:
I think we should extend `HvEmptyState` to enable the customisation of the empty state component without any limitations (icon, classes, etc):

```
export interface HvFlowEmptyProps extends HvEmptyStateProps { }
```

and then we can do something like:

```
import { useClasses } from "./Empty.styles";

export const HvFlowEmpty = ({ className, ...others }: HvFlowEmptyProps) => {
   const { classes, css } = useClasses();
   
   const nodes = useStore((state) => state.getNodes());
   
   return (
     !nodes ||
     (nodes.length === 0 ? (
       <HvEmptyState 
            className={cx(classes.root, className)}
            {...others} 
       />
     ) : null)
   );
 };
```

```
// Empty.styles.tsx

export const { staticClasses, useClasses } = createClasses("HvFlowEmpty", {
  root: {
     width: "100%",
     height: "100%",
     alignItems: "center",
     justifyContent: "center",
     backgroundColor: theme.colors.backgroundColor,
     position: "absolute",
  },
});

``` 

I'm also using the `css` util from `useClasses` to use our emotion cache.

Reply:
Yep, that's a good idea, thanks.